appAndroid/post/src/androidTest/java/com/gchristov/newsfeed/post/PostTest.kt

```diff
@@ -157,3 +167,28 @@ private const val PostTitle = "Post Title"
 private const val PostAuthor = "Anonymous"
 private const val PostReadingTime = "1 min read"
 private const val PostBody = "This is a sample post body"
+
+// 549 words ~= 3 minutes
```

Comment:
These don't appear to be used anywhere?

I think we have two options:
1. we either have UI tests which test different post bodies and expect the correct reading time to be displayed
2. or we have specialised unit tests for this testing the ReadingTimeCalculator in isolation

In general we want as much integration tests as possible because this is the closest we can get to a real user experience - unit tests only cover a tiny part of the interaction. 

Because of this, I would go with option 1 above and:
• refactor PostCreator to take in a `body`
• within the UI tests pass the body we want and expect the correct reading time to be shown on the UI - e.g. we pass the short title above and we should expect 1 min read, then we pass the long title and we expect 3 min read, then pass empty title+body and expect 0 reading time

This approach basically covers all user-facing cases so we would be confident that what they see is correct, whereas with the unit test we're still not 100% confident as we'd have to cover the UI+view-model part separately.

Let me know if this needs more explanation - it was new to me too and I learned it in Monzo - we always make sure our UI tests cover as much as possible 👍

Reply:
I was adding a post with a body of `549` words which would've shown `3 minutes read` :)

Reply:
Created #6. I think it's fine to address in a separate PR in the interest of keeping changes concise and small. I vote for removing the constants from here for now and re-add them to the next tests PR. What do you think?

Reply:
Agreed, I'll remove them and you can close this comment :)